airflow/serialization/schema.json

```diff
@@ -161,6 +161,7 @@
         "weight_rule": { "type": "string" },
         "executor_config": { "$ref": "#/definitions/dict" },
         "do_xcom_push": { "type": "boolean" },
+        "xcom_key": { "type": "string" },
```

Comment:
This attribute isn't needed for webserver or scheduler, so we should exclude it from the serialization.

Reply:
I think that we need it since it's an additional field in the base operator. It's actually a test failure that pointed me to it